### ISSUE TITLE
runSCENIC_3_scoreCells error

### ISSUE THREAD

**Issue by alisyedraza1** [aertslab/SCENIC#427]
Hi all, 

I am running SCENIC pipeline, and get stuck specifically on runSCENIC_3_scoreCells function with the following error:

Error in par(plt = gridPLT(), new = TRUE) : 
  invalid value specified for graphical parameter "plt"

Could anyone please help regarding this error? Thanks. 

Best, 
Ali.

**Comment by Biomamba** [comment]
Cloud U please tell me if you solved the problem? I met the same error.

**Comment by alisyedraza1** [comment]
Yes, I did figure it out. You have to switch the type of the input object
to matrix form. I have forgotten exactly at what point but I can check and
let you know. Then it will run fine. Thanks.

Best,
Ali.
